aviary/interface/methods_for_level2.py

```diff
@@ -1650,7 +1650,7 @@ def add_design_variables(self):
             optimize_mass = self.pre_mission_info.get('optimize_mass')
             if optimize_mass:
                 self.model.add_design_var(Mission.Design.GROSS_MASS, units='lbm',
-                                          lower=100.e3, upper=200.e3, ref=135.e3)
+                                          lower=100.e2, upper=200.e3, ref=135.e3)
```

Comment:
I think eventually we want to allow the user to set these through the interface, though we probably would still have default bounds and refs, and these are better values.

(Note: IPOPT likes tighter bounds in general.)